extensions-contrib/opentelemetry-emitter/pom.xml

```diff
@@ -75,7 +75,23 @@
     </dependency>
     <dependency>
       <groupId>io.opentelemetry</groupId>
-      <artifactId>opentelemetry-exporter-otlp</artifactId>
```

Comment:
any reason this was removed?

Reply:
https://github.com/confluentinc/druid/pull/61